src/components/Login/index.jsx

```diff
@@ -0,0 +1,117 @@
+import React from "react";
+import { Redirect } from "react-router-dom";
+
+export default class Login extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isAuthenticated: false,
+      isShowingError: false,
+      userName: "",
+      password: "",
+    };
+
+    this.handleLogin = this.handleLogin.bind(this);
+  }
+
+  updateField(e, field) {
+    this.setState({
+      [field]: e.target.value,
+    });
+  }
+
+  handleLogin() {
+    const { userName, password } = this.state;
+
+    if (userName === "ziv" && password === "papa") {
```

Comment:
perhaps create an object that contains a bunch of accepted username/password combos, that way more than this specific user can log in